.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,31 @@
+name: Feature Request
+description: Suggest a new feature for Juju
+labels:
+  - kind/feature
+  - needs-triage
+body:
+  - type: textarea
+    id: Problem-Description
+    attributes:
+      label: Problem description
+      description: "Describe the problem/use-case: "
+      placeholder: ex. "I need to be able to see the local time on each machine."
+    validations:
+      required: true
+
+  - type: textarea
+    id: Desired-Solution
+    attributes:
+      label: Desired solution
+      description: "Describe an acceptable solution: "
```

Comment:
Not sure we want this section. Users are notoriously bad at specifying what any solution should be. Can we just ask them to describe their problem and leave it at that?

Reply:
In general I do believe we need the option to report a feature request. 
Having the option to request a feature, challenges the reporter to think if what they're asking is a bug or not. 
I would like this to be rephrase the "Desires solution" to "Desired behavior" or something of the sort. A feature should not be presented as a problem to be solved, but as an opportunity to improve.

Reply:
I went with "Desired behaviour".